lib/sockjs.js

```diff
@@ -547,19 +547,23 @@
     return new InfoAjax(url, XHRFake);
   };
 
-  InfoReceiver.prototype.doXhr = function(baseUrl, urlInfo) {
+  InfoReceiver.prototype.doXhr = function(baseUrl, urlInfo, timeout) {
     var self = this
       , url = urlUtils.addPath(baseUrl, '/info')
       ;
     debug('doXhr', url);
 
     this.xo = InfoReceiver._getReceiver(baseUrl, url, urlInfo);
 
+
+    if (timeout === undefined) {
+      timeout = InfoReceiver.timeout
```

Comment:
> If sockJS's internally calculated
>  /// timeout is higher, that will be used instead.
😕 if true, shouldn't this conditional be < ? But maybe also set a lower limit?

Reply:
Ah, it's *min* timeout!

Reply:
Oh, whoops, I must have had that slip through the cracks somewhere.  I'll see what the sockjs maintainer says.